README.md

```diff
@@ -19,9 +19,22 @@ cargo test
 
 ### Running the Examples
 
-First, ensure you have a local MQTT broker running, such as [Mosquitto](https://github.com/eclipse/mosquitto).
+1. Start an MQTT broker (e.g. mosquitto)
+
+2. Set up your environment (for example with a config file at .cargo/config.toml)
+
+Make sure to set these parameters:
+```toml
+[env]
+MQTT_PROTOCOL = "'mqtt' or 'mqtts'"
+MQTT_PORT = "8883 for ssl encrypted mqtt"
+MQTT_HOSTNAME = "the hostname/ url of the broker"
```

Comment:
FMPOV this could be replaced by a single param (e.g. `MQTT_SERVER_URI`) which contains the URI to connect to the server.

Reply:
Not opposed to this. Did not change it till now since it doesnt relate to mqtts/ mqtt. Alternatively we could also assume that the mqtt -> 1883 and mqtts -> 8883 mapping always holds true and get rid of the port that way?